apps/web/vibes/soul/sections/footer/index.tsx

```diff
@@ -74,22 +78,30 @@ export const Footer = forwardRef(function Footer(
             >
               {(logo) => {
                 if (logo != null && typeof logo === 'string') {
-                  return (
+                  return logoHref !== undefined && logoHref !== '' ? (
                     <Link
+                      aria-label={logoLabel}
                       className="relative mb-2 inline-block h-10 w-full max-w-56 rounded-lg ring-primary focus-visible:outline-0 focus-visible:ring-2"
-                      href="#"
+                      href={logoHref}
                     >
                       <span className="whitespace-nowrap font-heading text-2xl font-semibold">
                         {logo}
                       </span>
                     </Link>
+                  ) : (
+                    <div className="relative mb-2 inline-block h-10 w-full max-w-56 rounded-lg ring-primary focus-visible:outline-0 focus-visible:ring-2">
+                      <span className="whitespace-nowrap font-heading text-2xl font-semibold">
+                        {logo}
+                      </span>
+                    </div>
```

Comment:
What if instead of duplicating the content we default the `href` to `#`?

Reply:
Fixed on latest commit.